### DISCUSSION TITLE
Hi, I need to know if it is possible and what is the best configuration

### DISCUSSION THREAD

**Discussion by surhack** [Chia-Network/chia-blockchain#4581] [Support]
I am interested in chia blockchain mining, I have 2 machines, 1 machine with i7 processor, 64 GB RAM, very fast M2. SSD's etc.. etc.. that i use for another projects, and a secondary old machine (with XEON, 8 Gb RAM)
My idea is make the plots in the first machine and farm the plots in the secondary machine
Both machines are in the same LAN (same router), and I would like to know if it is necesary open port 8444 to the first, the second, or both. And if it is necessary open to both, it is impossible to open the same port, Is is possible to change in the machine configuration of Chia software?
Both machines run over windows 10 last edition.

Thanks in advanced.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
